src/converters/account.ts

```diff
@@ -16,21 +17,33 @@ import {
 export function createAccountEntity(
   config: OktaIntegrationConfig,
   data: OktaAccountInfo,
+  supportData?: OrgOktaSupportSettingsObj,
 ): StandardizedOktaAccount {
   let displayName = data.name;
   if (data.preview) {
     displayName += ' (preview)';
   }
 
   const accountId = config.oktaOrgUrl.replace(/^https?:\/\//, '');
-
+  let supportEnabled: boolean | null = null;
+  let supportExpires: number | undefined = undefined;
+  if (supportData) {
+    if (supportData.support === 'ENABLED') {
+      supportEnabled = true;
+    } else {
+      supportEnabled = false;
+    }
+    supportExpires = parseTimePropertyValue(supportData.expiration);
+  }
   return {
     _type: Entities.ACCOUNT._type,
     _key: `okta_account_${accountId}`,
     _class: Entities.ACCOUNT._class,
     name: data.name,
     displayName,
     accountId,
+    supportEnabled: supportEnabled,
+    supportExpiresOn: supportExpires,
```

Comment:
`parseTimePropertyValue` defaults to `undefined if the variable passed in is `undefined`.

```suggestion
    supportExpiresOn: parseTimePropertyValue(supportData?.expiration),
```